app/src/main/res/layout/layout_pairing.xml

```diff
@@ -107,7 +107,7 @@
         app:srcCompat="@drawable/ic_arrow_back_24dp" />
 
     <TextView
-        android:id="@+id/layout_create_safe_intro_title"
+        android:id="@+id/layout_manage_tokens"
```

Comment:
yay AS renaming refactor ... should probably be something with `layout_pairing_titele`